newrelic_lambda_cli/integrations.py

```diff
@@ -501,6 +501,34 @@ def update_log_ingestion(
         return True
 
 
+def get_log_ingestion_license_key(session):
```

Comment:
Why are we doing this? Shouldn't GraphQL be the source of truth?

Reply:
In the upgrade scenario, we don't currently ask the user for an API key. It would be a pretty significant customer annoyance to do so. I don't think there are any situations where this won't work.